doc/main.dox

```diff
@@ -64,11 +64,14 @@ under the GPL.
 
 \section yarp_basics_usage The basics of using YARP
 \li \subpage yarp_cmake_hello
-\li \subpage tutorials
 \li \subpage note_ports
 \li \subpage note_devices
 \li \subpage yarp_data_dirs
 
+\section Learning YARP
+The best way to learn how to use YARP is to read the tutorials and documentation pages that are reported in the page \subpage tutorials. Do not forget to checkout the command line tools and guis sections below. Advanced topics for interoperability and technical papers are reported at the end of this page. The page \supage tutorials is continuously updated  so it is a good idea to periodically check it.
```

Comment:
There are two spaces in "[...] continuously **updated  so** it is a good idea [...]".